src/WireMock.Net/Settings/ProxyUrlReplaceSettings.cs

```diff
@@ -0,0 +1,17 @@
+namespace WireMock.Settings;
+
+/// <summary>
+/// Defines an old path param and a new path param to be replaced when proxying.
+/// </summary>
+public class ProxyUrlReplaceSettings

```

Comment:
Also this new class needs to be defined as a model in https://github.com/WireMock-Net/WireMock.Net/tree/master/src/WireMock.Net.Abstractions/Admin/Settings

And the values should be mapped from the internal model to this model class 9and the other way around)

Reply:
The `/src/WireMock.Net.Abstractions/Admin/Settings)` contain the (Json) Models which are used when you add / update / view the settings via the admin interface. (https://github.com/WireMock-Net/WireMock.Net/wiki/Admin-API-Reference)

In order to make add / update / view possible for these new settings, the internal class + your new class (src/WireMock.Net/Settings/ProxyAndRecordSettings.cs) needs to be mapped to that json model class, and the other way around.

But maybe this class takes care of that:
https://github.com/WireMock-Net/WireMock.Net/blob/master/src/WireMock.Net/Util/TinyMapperUtils.cs

Reply:
Ah, I understand now, thank you for clarifying that. I'll get the mapper updated asap

Reply:
@StefH updated with your recommendations.